prov/verbs/src/ep_rdm/verbs_utils.h

```diff
@@ -62,6 +62,7 @@ struct fi_ibv_msg_ep;
 
 #define FI_IBV_RDM_DFLT_ADDRLEN	(sizeof (struct sockaddr_in))
 
+#define FI_IBV_RDM_RESOLVE_ADDR_TIMEOUT (30000)
```

Comment:
There's already FI_IBV_RDM_CM_RESOLVEADDR_TIMEOUT below. Seems like the same thing.

Reply:
Do you mean `FI_IBV_RDM_CM_THREAD_TIMEOUT` ?
verbs uses `FI_IBV_RDM_CM_THREAD_TIMEOUT` to sleep between attempts for calling `rdma_get_cm_event` if it returns `EAGAIN`.

Reply:
I see
    #define FI_IBV_RDM_CM_RESOLVEADDR_TIMEOUT (30000)

Looks like the same to me.